pkg/controller/controller_binding.go

```diff
@@ -385,6 +413,7 @@ func (c *controller) reconcileServiceInstanceCredential(binding *v1alpha1.Servic
 					v1alpha1.ConditionFalse,
 					errorUnbindCallReason,
 					"Unbind call failed. "+s)
+				toUpdate.Status.OperationStartTime = nil
```

Comment:
I think that you should not clear the operation start time here, unless you are done doing work.

Reply:
I think that we are done doing work here. I think that the current behavior of returning the error here is incorrect. I think we should be falling through and continuing with the process of deleting the resource from the registry. We tried to delete the binding and the Broker responded with an error indicating that it could not delete the binding. We should give up at that point with trying to have the Broker delete the binding. We don't try again in this case for Instances.